python/src/GaussLegendre_doc.i.in

```diff
@@ -9,27 +9,100 @@ Available constructors:
 Parameters
 ----------
 dimension : int, :math:`dimension>0`
-    The dimension of the functions to integrate. The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
+    The input dimension of the functions to integrate.
 discretization : sequence of int
     The number of nodes in each dimension. The sequence must be non-empty and must contain only positive values.
+    
+    The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension,
+    see :class:`~openturns.ResourceMap`.
 
 Notes
 -----
-The Gauss-Legendre algorithm enables to approximate the definite integral:
+We consider a function :math:`f: \Rset^d \mapsto \Rset^p` and a domain :math:`\set{D} = [\vect{a}, \vect{b}]`. The Gauss-Legendre algorithm enables to approximate the definite integral:
```

Comment:
I suggest to 1) use the mathematical notations, 2) simplify the verbs and 3) define the dimension of the bounds of the interval:
```suggestion
We consider a function :math:`f: \Rset^\inputDim \mapsto \Rset^\outputDim` and a domain :math:`\set{D} = [\vect{a}, \vect{b}]` where :math:`\vect{a}, \vect{b} \in \Rset^\inputDim`. The Gauss-Legendre quadrature approximates the definite integral:
```

Reply:
ok